src/extension.ts

```diff
@@ -219,14 +219,19 @@ function launchMetals(
       }
       return doctor;
     }
-    ["build-import", "build-connect", "sources-scan", "doctor-run"].forEach(
-      command => {
-        const cancel = commands.registerCommand("metals." + command, async () =>
-          client.sendRequest(ExecuteCommandRequest.type, { command: command })
-        );
-        context.subscriptions.push(cancel);
-      }
-    );
+    [
+      "build-import",
+      "build-connect",
+      "sources-scan",
+      "doctor-run",
+      "compile-cascade",
+      "compile-cancel"
+    ].forEach(command => {
```

Comment:
Not strictly related to this PR, but I suspect something like this would work:

```typescript
JSON.parse(fs.readFileSync('package.json', 'utf8')).contributes.commands.forEach(({ command }) => {
 const [, commandName] = command.split('.');
 const cancel = commands.registerCommand(command, async () =>
   client.sendRequest(ExecuteCommandRequest.type, { command: commandName })
 );
 context.subscriptions.push(cancel);
})
```

Reply:
This is a good idea but it requires automatically converting from camel case in package.json to kebab case here. I think it's better left to a separate PR, I'm fine with the manual process.